envs/requirements-sklearn.txt

```
@@ -0,0 +1,21 @@
+# additional frameworks
+xgboost
+catboost
+lightgbm
+faiss-cpu
+modin[all]
+scikit-learn-intelex
+dpctl
+dpnp
+# sklbench dependencies
+scikit-learn==1.4.*
```

```suggestion
scikit-learn>=1.4.*
```
Ideally to reduce maintenance

---

Changed to `scikit-learn<1.5` since sklearnex version supporting sklearn 1.5 is not released yet.